tests/web/rpc/v1/test_endpoint_trace_item_table/test_endpoint_trace_item_table.py

```diff
@@ -183,9 +183,11 @@ def write_eap_span(
 @pytest.fixture(autouse=False)
 def setup_teardown(clickhouse_db: None, redis_db: None) -> None:
     spans_storage = get_storage(StorageKey("eap_spans"))
+    items_storage = get_storage(StorageKey("eap_items"))
     start = BASE_TIME
     messages = [gen_message(start - timedelta(minutes=i)) for i in range(120)]
     write_raw_unprocessed_events(spans_storage, messages)  # type: ignore
+    write_raw_unprocessed_events(items_storage, messages)  # type: ignore
```

Comment:
I would extract the double writing to its own function and change that across the board rather than writing the double write everywhere